playbooks/roles/ocp-cli-plugins/README.md

```diff
@@ -0,0 +1,101 @@
+|--------------------------------|----------|-----------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| latest_go_version_tar          | yes      | https://go.dev/dl/go1.18.4.linux-ppc64le.tar.gz                                                                 | Golang latest version tar URL to install latest golang for the test, default tar is for go version- 1.18.4                                                                           |
+| enable_cli_plugins_test        | no       | false                                                                                                           | Flag to be set to true to enable custom CLI-plug-in test playbook                                                                                                                    |
+| deploy_cli_plugins             | yes      | empty list                                                                                                      | List of cli-plugin names to be deployed and tested, plugin names must be among ["custom","sample","krew","mirror","compliance"]. Empty list will not deploy or test any cli-plugin   |
+| user_custom_plugin             | no       | empty string                                                                                                    | File name of user defined custom-cli-plugin prefixed with "oc-" e.g. user_custom_plugin: "oc-test-plugin.txt"                                                                        |
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+
+
+Steps to deploy and test the CLI-plug-ins via playbook
+-------------------------------------------------------
+
+1. Custom CLI-plug-in: This CLI-plug-in can be tested with default OR user-defined CLI-plug-in. Following are the steps to test the user-defined custom CLI-plug-in.
+
+    - Add the name "custom" into the list variable deploy_cli_plugins mentioned above in the Role Variables
+    - Create a shell/bash file of your custom CLI-plug-in with the prefix "oc-" in the name e.g. oc-test-plugin.txt
+    - Place at role path ~/ocp4-playbooks-extras/playbooks/roles/ocp-cli-plugins/files/
+    - Assign the plug-in name to variable "user_custom_plugin" in the var file cli_plugins_vars.yaml
+    - Follow the below steps to run the playbook
+    
+2. Sample CLI-plug-in: This CLI-plug-in test requires golang version>=1.18 installed on the cluster. Follow the below steps to enable sample-cli-plugin test
```

Comment:
Sample CLI plugin is only used for test purposes. I suppose this can be omitted.